dist/src/test/java/io/camunda/application/commons/console/ping/PingConsoleConfigurationTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.application.commons.console.ping;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.application.commons.console.ping.PingConsoleConfiguration.ConsolePingConfiguration;
+import io.camunda.service.ManagementServices;
+import io.camunda.service.license.LicenseType;
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class PingConsoleConfigurationTest {
+
+  @Mock private ManagementServices managementServices;
+  @Mock private ConsolePingConfiguration pingConfiguration;
+  @Mock private HttpClient mockClient;
+
+  @Test
+  void shouldFailToStartConsolePing() {
```

Comment:
🙃 I'd suggest to split these up into separate tests, each with the usual `given/when/then` style